capture/SharpCap/samples/Totality.py

```diff
@@ -6,107 +6,80 @@
 #           
 # Authors:  Chris Mandrell
 #           Matt Penn
+#           Castor Fu
 # Written: 3/20/2023
-# Last Edit: 4/4/2023 CM
+# Last Edit: 5/15/2023 CM
```

Comment:
Sorry, I should have said this earlier.  Since the revision control system tracks this stuff, there's no need to clutter up the code with it.  Please delete.